### PR TITLE
Add ARM template schema for Kusto 2018-09-07 preview

### PR DESCRIPTION
**Pull request by radennis**
Add ARM template schema for Kusto 2018-09-07 preview

### CODE REVIEW

File: schemas/2015-01-01/deploymentTemplate.json [hunk 1]
```diff
@@ -144,6 +144,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2014-04-01-preview/Microsoft.Sql.json#/resourceDefinitions/disasterrecoveryconfiguration" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-07-privatepreview/Microsoft.Kusto.json#/resourceDefinitions/clusters" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-07-privatepreview/Microsoft.Kusto.json#/resourceDefinitions/clusters_databases" },
+				  { "$ref": "https://schema.management.azure.com/schemas/2018-09-07-preview/Microsoft.Kusto.json#/resourceDefinitions/clusters" },
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: >				 [](start = 0, length = 4)

please use spaces.

File: schemas/2018-09-07-preview/Microsoft.Kusto.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
+﻿{
+  "id": "https://schema.management.azure.com/schemas/2018-09-07-preview/Microsoft.Kusto.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Kusto",
+  "description": "Microsoft Kusto Resource Types",
+  "resourceDefinitions": {
+    "clusters": {
```
Comment (TianoMS) [inline schemas/2018-09-07-preview/Microsoft.Kusto.json]: >clusters [](start = 5, length = 8)

please add tests for the new resource types.

File: schemas/2015-01-01/deploymentTemplate.json (hunk 1 above)
Comment (radennis) [reply schemas/2015-01-01/deploymentTemplate.json]: Done

File: schemas/2018-09-07-preview/Microsoft.Kusto.json (hunk 2 above)
Comment (radennis) [reply schemas/2018-09-07-preview/Microsoft.Kusto.json]: Done

**Comment by radennis** [comment]
@TianoMS ping

**Comment by radennis** [comment]
@TianoMS  could you please take a look?

**Pull request merged**